readme.md

```diff
@@ -37,8 +38,10 @@ composer require brokerexchange/showcase
 ```
 
 To include the assets, you have two options:
-    1. `@import "vendor/brokerexchange/showcase/resources/assets/build/public";` in your main stylesheet, which will include the styles as part of your app's compiled styles.
-    2. `php artisan vendor:publish --tag=showcase-assets` and link to `/vendor/showcase/public.css` in the `<head>`.
+
```

Comment:
I meant for this to be a list, so it should be removing the spaces before the numbers.

Reply:
spaces make it an ordered list `<ol>`